bindings/core/tests/combined.rs

```diff
@@ -68,113 +49,114 @@ async fn create_account() -> Result<()> {
-        .with_coin_type(SHIMMER_COIN_TYPE)
-        .with_secret_manager(serde_json::from_str::<SecretManagerDto>(secret_manager).unwrap())
-        .build()
-        .await?;
-
-    let mut account_details = BTreeMap::new();
-    let mut handle_response = |response| match response {
-        Response::Account(account) => {
-            account_details.insert(account.index, account);
-        }
-        _ => panic!("unexpected response {response:?}"),
-    };
-
-    // Create a few accounts
-    for alias in ["Alice", "Bob", "Roger", "Denise", "Farquad", "Pikachu"] {
-        handle_response(
-            wallet
-                .call_method(WalletMethod::CreateAccount {
-                    alias: Some(alias.to_owned()),
-                    bech32_hrp: None,
-                    addresses: None,
-                })
-                .await,
-        );
-    }
-
-    // Remove latest account
-    match wallet.call_method(WalletMethod::RemoveLatestAccount).await {
-        Response::Ok => {}
-        response => panic!("unexpected response {response:?}"),
-    }
-
-    account_details.pop_last();
-
-    // Get individual account details
-    for account in account_details.values() {
-        // By Index
-        match wallet
-            .call_method(WalletMethod::GetAccount {
-                account_id: account.index.into(),
-            })
-            .await
-        {
-            Response::Account(details) => {
-                assert_eq!(&account_details[&details.index], &details);
-            }
-            response => panic!("unexpected response {response:?}"),
-        }
-
-        // By Name
-        match wallet
-            .call_method(WalletMethod::GetAccount {
-                account_id: account.alias.as_str().into(),
-            })
-            .await
-        {
-            Response::Account(details) => {
-                assert_eq!(&account_details[&details.index], &details);
-            }
-            response => panic!("unexpected response {response:?}"),
-        }
-    }
-
-    // Get account details
-    match wallet.call_method(WalletMethod::GetAccounts).await {
-        Response::Accounts(details) => {
-            assert_eq!(account_details.len(), details.len());
-            for detail in details {
-                assert_eq!(&account_details[&detail.index], &detail);
-            }
-        }
-        response => panic!("unexpected response {response:?}"),
-    }
-
-    // Get account indexes
-    match wallet.call_method(WalletMethod::GetAccountIndexes).await {
-        Response::AccountIndexes(indexes) => {
-            assert_eq!(account_details.len(), indexes.len());
-            for index in indexes {
-                assert!(account_details.contains_key(&index));
-            }
-        }
-        response => panic!("unexpected response {response:?}"),
-    }
-
-    std::fs::remove_dir_all(storage_path).ok();
-    Ok(())
-}
+// TODO: temporarily disabled (needs to become `verify_wallet`)
```

Comment:
Should be fine to just remove, this was there to make sure we don't mess the account indexes up, but this can't happen anymore

Reply:
removed